ops/packer/build_bfd-all.json

```diff
@@ -58,6 +58,33 @@
         "Branch": "{{user `git_branch`}}",
         "Commit": "{{user `git_commit`}}"
       }
+    },
+    {
+      "name": "bfd-db-migrator",
+      "type": "amazon-ebs",
+      "iam_instance_profile": "bfd-packer",
+      "ssh_username": "ec2-user",
+      "instance_type": "m5.xlarge",
```

Comment:
non-blocking question -- do we need this size of an instance?

Reply:
To be clear, this is the instance type that's created to serve as the source of the resultant AMI by the _packer builder_ `amazon-ebs` in the following, summarized process:
1. An ec2 instance of type `m5.xlarge` is created via the `amazon-ebs` _packer builder_
2. The routines encoded in the `ansible` _packer provisioner_  run against this instance
3. An AMI is generated from the instance

More on this builder can be found [here](https://www.packer.io/plugins/builders/amazon/ebs).